test/test_audio_manager.js

```diff
@@ -0,0 +1,131 @@
+require('jsdom-global')()
```

Comment:
Test classes fulfil requirement 4 - greater than 90% coverage in all aspects achieved.